ironfish/src/network/utils/serializers.ts

```diff
@@ -19,26 +20,26 @@ const BLOCK_TRANSACTIONS_LENGTH_BYTES = 2
 
 export function writeBlockHeader(
   bw: bufio.StaticWriter | bufio.BufferWriter,
-  header: SerializedBlockHeader,
+  header: BlockHeader,
```

Comment:
I'd probably double-check that the serialization didn't accidentally change, since a lot of these fields are swapping types. I think you could do this pretty easily by adding a log to the GetBlock test that prints the output, then running it on both staging and this branch and checking that they're equal.

Or you could connect an older version of ironfish to a node running this branch and make sure they can communicate, but that's probably a bit more time-consuming.

Reply:
Checked and confirmed that the serialization hasn't changed